pkg/kernel/tools/heal/liveness_check.go

```diff
@@ -55,16 +55,21 @@ func KernelLivenessCheck(deadlineCtx context.Context, conn *networkservice.Conne
 
 	var pinger *ping.Pinger
 
+	srcAddr, _, err := net.ParseCIDR(conn.GetContext().GetIpContext().GetSrcIpAddrs()[0])
+	if err != nil {
+		log.FromContext(deadlineCtx).Errorf("ParseCIDR for source address failed: %s", err.Error())
+		return false
```

Comment:
1. What if we have more than one address?
2. Can we use the helper function `GetSrcIPNets()`?

Reply:
Fixed